packs/agents-of-edgewatch-bestiary/alchemist-aspirant.json

```diff
@@ -357,7 +357,7 @@
                         ],
                         "selector": "{item|_id}-attack",
                         "text": "PF2E.BombNotes.TanglefootBag.Moderate.success",
-                        "title": "ITEM.TypeEffect"
+                        "title": "TYPES.Item.effect"
```

Comment:
This should have been migrated by Migration788UpdateTanglefootBags, but perhaps it wasn't run on compendium items?